Library/Formula/vault.rb

```diff
@@ -0,0 +1,35 @@
+require "language/go"
+
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  version "0.1.0"
+  url "https://github.com/hashicorp/vault.git"
```

Comment:
Unfortunately the upstream requires the `.git` directory to be intact (which the tarball doesn't have) so I'm using git instead.  I've now added the `:tag`.